src/sonic-build-hooks/scripts/buildinfo_base.sh

```diff
@@ -28,8 +28,7 @@ else
 	PKG_CACHE_PATH=/sonic/target/vcache/${IMAGENAME}
 fi
 PKG_CACHE_FILE_NAME=${PKG_CACHE_PATH}/cache.tgz
-$SUDO mkdir -p ${PKG_CACHE_PATH}
-$SUDO chown $USER $PKG_CACHE_PATH
```

Comment:
From the log, it looks like $USER="".
But it needs tests. SUDO can't be removed. Because there will be other error message.

Reply:
@liushilongbuaa originally it was without `sudo`. Please see the PR description. Looks like `sudo` is not needed since the cache path is created with the regular user privileges long before executing the script during docker container build (environment doesn't have `USER` variable defined)

Reply:
Building sonic-slave-* is different from build target/sonic-*.gz.
Some needs sudo, but some din't need.
I added sudo because of error message.